content/resources/lessons/sound-basics/presentation/index.md

```diff
@@ -166,6 +214,11 @@ $$
 
 Maximum frequency in spectrograms = sample rate / 2 = <mark>Nyquist</mark>
 
+>Notes:
+> - The maximum frequency in a spectrogram is the sample rate divided by 2. This
+>   is also known as the Nyquist frequency.
```

Comment:
putting the cart before the donkey.

Explain the Nyquist theorem a little and that will explain why maximum affective frequency is half of the sample rate.